src/main/java/com/eppo/sdk/EppoClient.java

```diff
@@ -68,96 +52,154 @@ private Optional<EppoValue> getAssignmentValue(
         InputValidator.validateNotBlank(subjectKey, "Invalid argument: subjectKey cannot be blank");
         InputValidator.validateNotBlank(flagKey, "Invalid argument: flagKey cannot be blank");
 
+        VariationAssignmentResult assignmentResult = this.getAssignedVariation(flagKey, subjectKey, subjectAttributes);
+
+        if (assignmentResult == null) {
+            return Optional.empty();
+        }
+
+        Variation assignedVariation = assignmentResult.getVariation();
+        Optional<EppoValue> assignmentValue = Optional.of(assignedVariation.getTypedValue());
+
+        // Below is used for logging
+        String experimentKey = assignmentResult.getExperimentKey();
+        String allocationKey = assignmentResult.getAllocationKey();
+        String assignedVariationString = assignedVariation.getTypedValue().stringValue();
+
+        if (assignedVariation.getAlgorithmType() == AlgorithmType.OVERRIDE) {
+            // Assigned variation was from an override; return its value without logging
+            return assignmentValue;
```

Comment:
For now, I'll keep supporting overrides as long as the RAC generation code supports them (even if it's not in our UI/UX). Seems like the safe thing to do.